common/pom.xml

```diff
@@ -165,6 +165,22 @@
                     <groupId>com.sun</groupId>
                     <artifactId>tools</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.mortbay.jetty</groupId>
+                    <artifactId>servlet-api-2.5</artifactId>
```

Comment:
Alright, cobertura-2.0.3 depends on servlet-api-2.5(org.mortbay.jetty), hence the exclusion.